docs/source/filters.rst

```diff
@@ -86,7 +86,7 @@ SimpleITK Filters
      - BinaryThreshold projection.
```

Comment:
I believe this file may be generated from "Utilities/filters.csv" with the "CompareITKandSITKFilters.py". That script looks like it is in need of an update. Some of the incorrect filter names in the simpleITK doxygen linked may be due to issues in the script.

Additionally, I am not sure we should change the simpleitk URL from latest to different version numbers.

Reply:
All versions set to latest (force pushed). I think we can update the script at a later time.